Name: Reset Options On Power Button Hold
Version: 1.3.5-3
Author: Jason Robitaille
Description: On power button hold, it goes straight to the reset options, with airplaine mode added, and skips the swap-battery/shutdown prompt,   (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -2426,7 +2426,7 @@
 		    var f = function(stageController){
 		      stageController.pushScene('poweroffalert',this);
 		    }.bind(this);
-		    var params = {name: 'PowerOffAlert', height: 175,lightweight:true};		    	    
+		    var params = {name: 'PowerOffAlert', height: 230,lightweight:true};
 		    appController.createStageWithCallback(params,f,'popupalert');
         }
 	//Close the Storage Alert and create a USB dashboard.
--- .orig/usr/lib/luna/system/luna-systemui/app/views/poweroffalert/poweroffalert-scene.html
+++ /usr/lib/luna/system/luna-systemui/app/views/poweroffalert/poweroffalert-scene.html
@@ -1,5 +1,6 @@
 <div class="system-notification power-off">
 		<div class="palm-notification-button affirmative" x-mojo-tap-highlight="immediate" id='flightmode' x-mojo-loc=''></div>
-		<div class="palm-notification-button negative" x-mojo-tap-highlight="immediate" id='poweroff' x-mojo-loc=''>Power</div>
+		<div class="palm-notification-button alternate" x-mojo-tap-highlight="immediate" id='restart' x-mojo-loc=''>Restart</div>
+		<div class="palm-notification-button negative" x-mojo-tap-highlight="immediate" id='shutdown' x-mojo-loc=''>Shut Down</div>
 		<div class="palm-notification-button" x-mojo-tap-highlight="immediate" id='cancel' x-mojo-loc=''>Cancel</div>
 </div>
\ No newline at end of file
--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/poweroffalert-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/poweroffalert-assistant.js
@@ -7,7 +7,8 @@
     
     setup: function() {		
 		var flightMode = this.flightMode.bindAsEventListener(this);
-		var powerOff = this.powerOff.bindAsEventListener(this);
+		var restart = this.restart.bindAsEventListener(this);
+		var shutdown = this.shutdown.bindAsEventListener(this);
 		var cancel = this.cancel.bindAsEventListener(this);
 		var launchAirplaneModeHelp = this.launchAirplaneModeHelp.bindAsEventListener(this);
 		this.flightMode = this.barAssistant.getAirplaneMode();
@@ -16,7 +17,8 @@
 		else
 			this.controller.get('flightmode').innerHTML = "<div class='info-icon' id='airplaneInfo' x-mojo-touch-feedback='immediate'></div>" + $L('Airplane Mode');
 			
-		this.controller.get('poweroff').addEventListener(Mojo.Event.tap, powerOff);
 		this.controller.get('flightmode').addEventListener(Mojo.Event.tap, flightMode);
+		this.controller.get('restart').addEventListener(Mojo.Event.tap, restart);
+		this.controller.get('shutdown').addEventListener(Mojo.Event.tap, shutdown);
 		this.controller.get('cancel').addEventListener(Mojo.Event.tap, cancel);
 		this.controller.get('airplaneInfo').addEventListener(Mojo.Event.tap, launchAirplaneModeHelp);
@@ -46,6 +48,16 @@
 		this.controller.window.close();
 	},
 	
+	restart: function() {
+		this.barAssistant.machineReboot("Restart selected by User in PowerOff Dialog");
+		this.controller.window.close();
+	},
+	
+	shutdown:function() {
+		this.barAssistant.machineOffNow("Shutdown request by User");
+		this.controller.window.close();
+	},
+	
 	flightMode:function(event) {
 		
 		var targetId = event.target.id;
